src/etc/inc/interfaces.inc

```diff
@@ -3196,7 +3196,35 @@ function interface_dhcpv6_configure($interface = 'wan', $wancfg)
     unset($dhcp6cconf);
 
     $dhcp6cscript = "#!/bin/sh\n";
+    $dhcp6cscript .= "dmips=\${new_domain_name_servers}\n";
+    $dhcp6cscript .= "dmnames=\${new_domain_name}\n";
```

Comment:
unused :)

Reply:
Yes, sorry I was playing around with those, forgot they were still there...

Reply:
I will attempt to remove them and issue a new commit, this is where it all went pear shaped last time.... here goes.